cdn_lambda/functions/map_to_s3.py

```diff
@@ -0,0 +1,37 @@
+from datetime import datetime
+
+from chexus import Client, TableItem
```

Comment:
I didn't know that you intend to integrate chexus here, I thought that was targeting only the publish side.

If you want to use chexus here, can you please donate it to release-engineering organization first, nominate CODEOWNERS, start using code review etc?  I would have asked for that around Phase II originally, but if you want to use it in the lambdas it should happen earlier.

Reply:
I think we don't need to use chexus here, but I've considered the possibility of transferring it to release-engineering.

If the code does go to release-engineering, I would prefer creating a new project there, initializing it with the current chexus code and prehaps changing the name, as I was never all that fond of the checksum-nexus portmanteau.